ansible/roles/apt_cacher_ng/defaults/main.yml

```diff
@@ -201,12 +201,8 @@ apt_cacher_ng__max_download_speed_kib: ''
 # One mirror per line.
 # Set to an empty string to let the package scripts from Apt-Cacher NG decide
 # which upstream mirror to use.
-apt_cacher_ng__upstream_mirror_debian: '{{ ansible_local.apt.default_sources_map.Debian[0]
-                                           if (ansible_local|d() and ansible_local.apt|d() and
-                                               ansible_local.apt.default_sources_map|d() and
-                                               ansible_local.apt.default_sources_map.Debian|d() and
-                                               ansible_local.apt.default_sources_map.Debian[0]|d())
-                                           else "http://deb.debian.org/debian" }}'
+apt_cacher_ng__upstream_mirror_debian: '{{ ansible_local.apt.default_sources_map.Dsdfebian[0]
```

Comment:
Spelling mistake (`Dsdfebian`).

Reply:
That was me testing if `d()` also works with array access and then forgetting to change it back before committing (and failing to self review my own commit changes). Thanks for your careful review!